### PR TITLE
feat: add Simple Card Entry form

### LINKED ISSUE #35: Add support for Square's single element payment form
https://developer.squareup.com/docs/payment-form/payment-form-walkthrough#build-a-single-element-payment-form-beta

### PR DESCRIPTION
**Pull request by maxbeatty**
fixes #35 

preview: https://react-square-payment-form-demo.maxbeatty.now.sh/

This approach requires a developer to create two instances of `SquarePaymentForm` if they want to use both `SimpleCard` _and_ digital wallets like Apple Pay which could be better. The upside is that this is a relatively small addition.

### CODE REVIEW

File: docs/simplecard.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
```
Comment (stephtaylor) [inline docs/simplecard.md]: You need to add `simplecard` to `sidebars.json` in order for it to render on the sidebar.

File: docs/simplecard.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+id: simplecard
+title: Single-Element Payment Form
+---
+
+[Build a single-element payment form (Beta)](https://developer.squareup.com/docs/payment-form/payment-form-walkthrough#build-a-single-element-payment-form-beta)
+
+Using the `SimpleCard` element is an alternative to building a payment form if you don't need the flexibility of individually placed payment card fields. [Read more about customizing the appearance](https://developer.squareup.com/docs/payment-form/cookbook/customize-form-styles#customize-the-appearance-of-the-single-element-payment-form).
```
Comment (stephtaylor) [inline docs/simplecard.md]: Could we add something about needing to complete steps 1-5 here https://square.github.io/react-square-payment-form/docs/paymentform? I'm worried about people jumping to this page and getting stuck if they copy-paste.

File: docs/simplecard.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        </div>
+      </SquarePaymentForm>
+    )
+  }
+}
+```
+
+## Caveats
+
+_You can **NOT** use digital wallets with `SimpleCard`._ If you would also like to accept payments with Apple Pay for Web, Google Pay, and Masterpass, you'll need to create a separate instance of `SquarePaymentForm` with digital wallet configuration.
```
Comment (stephtaylor) [inline docs/simplecard.md]: Nice addition!

**Comment by rohanbhangui** [comment]
@maxbeatty looks great thank you for building this so fast!

File: docs/simplecard.md (hunk 2 above)
Comment (rohanbhangui) [reply docs/simplecard.md]: @stephtaylor would be very helpful to have that!

**Pull request merged**